docs/CLI.md

```diff
@@ -11,50 +11,20 @@ Starting with v5 of WebdriverIO the testrunner will be bundled as a separate NPM
 npm install @wdio/cli
 ./node_modules/.bin/wdio --help
 
-WebdriverIO CLI runner
-
-Usage: wdio [options] [configFile]
-Usage: wdio config
-Usage: wdio repl <browserName>
-Usage: wdio install <type> <name>
-
-config file defaults to wdio.conf.js
-The [options] object will override values from the config file.
-An optional list of spec files can be piped to wdio that will override configured specs.
-Same applies to the exclude option. It can take a list of specs to exclude for a given run
-and it also overrides the exclude key from the config file.
+wdio <command>
 
 Commands:
-  wdio.js repl <browserName>  Run WebDriver session in command line
-  wdio.js install <type> <name> Add a `reporter`, `service`, or `framework` to your WebdriverIO project
+  wdio config                 Initialize WebdriverIO and setup configuration in
+                              your current project.
+  wdio install <type> <name>  Add a `reporter`, `service`, or `framework` to
+                              your WebdriverIO project
+  wdio repl <browserName>     Run WebDriver session in command line
+  wdio run <configPath>       Run your WDIO configuration file to initialize
```

Comment:
is this a breaking change?

Reply:
Yep, but we have a deprecation warning [here](https://github.com/webdriverio/webdriverio/pull/4402/files#diff-7a7619b7da49bfc975df94ab8009ba14R26-R29)

Reply:
I'm fine if it works in old way :)